### ISSUE TITLE
Cannot write in textfield inside alert view controller

### ISSUE THREAD

**Issue by MDLCHXD** [calabash/calabash-ios#1228]
Hi all, I'm automating my app and it uses an alert view with a text field inside of it. Once the test needs to touch the text field, it ends up pressing the button on the right of the alert controller, just below the text field. The button happens to be 'Cancel' so the alert disappears, but the calabash test line goes green although nothing in the app was named, so the test fails right after that. 

I wrote some custom steps and realized it had to do with a specific line regarding keyboard handling (I think).

```
Then /^I name the geofence$/ do
  touch("textField marked: 'geo_text_field'")
  wait_for_keyboard
  keyboard_enter_text "geofence01"
end
```

I believe it has to do with the touch line. It touches too low for this edge case. 
Also, the button that shows the alert, also raises the keyboard. Could that cause calabash to not realize the keyboard is open, since it wasn't raised upon touch of the text field?

I'm using your latest version (again I think) 0.20.3
Xcode 8.1 
iOS version physical and simulator 10

Your product is a life saver. Keep up the great work! 

Question, when is calabash-android gonna get the console upgrade like iOS? Or did I not get the memo! ios's console is SICK.

**Comment by jmoody** [comment]
I cannot reproduce.  

Can you show a screenshot of the alert?  

What is the form factor (e.g. 4s, 5s, 6, 7 etc.)?

**Comment by MDLCHXD** [comment]
it presses the cancel button when touching the text field

**Comment by jmoody** [comment]
Duplicate of **calabash-ios 0.20.0(include 0.20.3) keyboard_enter_text not work in alert dialog input views** #1220

Please see the comments there.

The gist is that we think we have a fix that will be available in run-loop 2.2.3.

Please paste a screenshot of the UIAlert.
